packages/datadog-plugin-fetch/test/index.spec.js

```diff
@@ -226,7 +228,7 @@ describe('Plugin', function () {
           .assertSomeTraces(traces => {
             expect(traces[0][0].meta).to.have.property(ERROR_TYPE, error.name)
             expect(traces[0][0].meta).to.have.property(ERROR_MESSAGE, error.message || error.code)
-            expect(traces[0][0].meta).to.have.property(ERROR_STACK, error.stack)
+            expect(traces[0][0].meta).to.have.property(ERROR_STACK, util.inspect(error, { depth: 0 }))
```

Comment:
What if `error` is not an actual error object?

Reply:
See https://github.com/DataDog/dd-trace-js/pull/6776/files#diff-5490d33b84fdf38d10fd4f4460c01af269e82e9a95d9bda16aef99b9353102b7R89-R97

We check for it having a message property or if it is an instance of error to determine if we add the tags.
The object will be serialized as a whole, if it contains a stack property. If it would be a object with a message and a stack property, that would be serialized instead of only showing the stack property. That in itself seems beneficial for users one way or the other. If it does not have a stack property, that is kept as is.

Reply:
I discussed it with error tracking and I am going to handle it as separate properties for now.
They said we could have a look how it would be on staging, while it will likely break a couple of existing pretty printing.

So I am marking it as draft for now.